{
  "name": "cpe:/a:ibm:datapower_gateway:2018.4.1.12",
  "title": [
    {
      "text": "IBM DataPower Gateway 2018.4.1.12",
      "lang": "en-US"
    }
  ],
  "references": [
    {
      "text": "Advisory",
      "href": "https://exchange.xforce.ibmcloud.com/vulnerabilities/184439"
    },
    {
      "text": "Product",
      "href": "https://www.ibm.com/products/datapower-gateway"
    }
  ],
  "cpe_23_item": {
    "name": "cpe:2.3:a:ibm:datapower_gateway:2018.4.1.12:*:*:*:*:*:*:*"
  }
}
